app/controllers/pull-requests.js

```diff
@@ -78,7 +78,7 @@ export default class PullRequestsController extends Controller {
   }
 
   updateContributorsList(users) {
-    const contributorsList = users
+    let contributorsList = users
```

Comment:
Can we revert this change? (I'm guessing you might have used `let` to indicate that the `pop()` method will mutate `contributorsList`.)

```suggestion
    const contributorsList = users
```